rxandroidble/src/test/groovy/com/polidea/rxandroidble2/exceptions/BleCannotSetCharacteristicNotificationExceptionTest.groovy

```diff
@@ -22,7 +22,7 @@ class BleCannotSetCharacteristicNotificationExceptionTest extends Specification
 
         then:
         assert objectUnderTest.toString() ==
-                "com.polidea.rxandroidble.exceptions.BleCannotSetCharacteristicNotificationException: " +
+                "com.polidea.rxandroidble2.exceptions.BleCannotSetCharacteristicNotificationException: " +
```

Comment:
🤢 Better to have no tests than not working tests that you don't know they don't.